run.sh

```diff
@@ -11,6 +11,8 @@ ceremony --config e1-key.yaml
 ceremony --config e2-key.yaml
 ceremony --config r3-key.yaml
 ceremony --config r4-key.yaml
+ceremony --config r3-cross-csr.yaml
```

Comment:
Would it be worthwhile to add a step that signs the CSR using openssl? It would somewhat complicate the script I guess, but would show interop with non-Go tooling.

Reply:
I'd like to add that in a follow-on PR if that's alright?